src/assertions.ts

```diff
@@ -10,6 +10,7 @@ import yaml from 'js-yaml';
 import Ajv, { ValidateFunction } from 'ajv';
 import addFormats from 'ajv-formats';
 import { distance as levenshtein } from 'fastest-levenshtein';
+import { Parser } from 'node-sql-parser';
```

Comment:
Because `node-sql-parser` is a `peerDependency`, we should lazily import and instantiate it within the `baseType === 'is-sql'` conditional.  Otherwise promptfoo will crash for people who have not manually installed the peer dependency.

Reply:
updated!